packages/electron-webpack/src/targets/RendererTarget.ts

```diff
@@ -99,7 +99,7 @@ export class RendererTarget extends BaseRendererTarget {
     // not configurable for now, as in the electron-vue
     const customTemplateFile = path.join(configurator.projectDir, configurator.rendererTemplate)
     const HtmlWebpackPlugin = require("html-webpack-plugin")
-    const nodeModulePath = configurator.isProduction ? null : path.resolve(require.resolve("electron"), "..", "..")
+    const nodeModulePath = configurator.isProduction ? null : path.resolve(require.resolve("electron-webpack"), "..", "..")
```

Comment:
This actually needs an extra `".."`, because `require.resolve("electron-webpack")` has more path components:

```js
> require.resolve("electron")
'/tmp/test/node_modules/electron/index.js'
> path.resolve(require.resolve("electron"), "..", "..")
'/tmp/test/node_modules'
> require.resolve("electron-webpack")
'/tmp/test/node_modules/electron-webpack/out/main.js'
> path.resolve(require.resolve("electron-webpack"), "..", "..")
'/tmp/test/node_modules/electron-webpack'
```